docs/application/dotnet/samples/system/overview.md

```diff
@@ -155,6 +155,13 @@ for TD:
 			<p>This sample application demonstrates how to obtain the ultraviolet sensor data using <a href="https://samsung.github.io/TizenFX/latest/api/Tizen.Sensor.html" target="_blank">Tizen.Sensor</a>.</p>
 			</td>
 		</tr>
+		<tr>
+			<td><img alt="" height="180" src="media/w82workout.png" width="180"/></td>
+			<td>
+			<p><strong>(W) Workout</strong> [In progress]</p>
+			<p>This sample application demonstrates how to create application which utilizes HRM sensor and location to track workout using <a href="https://samsung.github.io/TizenFX/latest/api/Tizen.Sensor.html" target="_blank">Tizen.Sensor</a>.</p>
```

Comment:
@sgchoi5 Request you to incorporate slight change in the sentence. 
```suggestion
                       <p>This sample application demonstrates how to create an application that utilizes the HRM sensor and location to track workout using <a href="https://samsung.github.io/TizenFX/latest/api/Tizen.Sensor.html" target="_blank">Tizen.Sensor</a>.</p>

Reply:
Updated it! plz, review changes